src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -350,7 +350,7 @@ std::optional<std::string> hashStringSHA1(std::string& input)
         return std::nullopt;
     }
 
-    if (1 != EVP_DigestInit_ex(ctx, EVP_sha1(), NULL))
+    if (1 != EVP_DigestInit_ex(ctx, EVP_sha1(), nullptr))
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#modern-language-features-and-best-practices

Use C++ nullptr instead of NULL.